openfoam-interface/internal-interface/libs/sourceFiles/LESModel.H

```diff
@@ -0,0 +1,249 @@
+/*---------------------------------------------------------------------------*\
```

Comment:
Similar https://github.com/simphony/simphony-openfoam/pull/23/files#r30581273 , there are no differences between LESModel.H and the one found in OpenFoam 2.2.x ("OpenFOAM-XXX/src/turbulenceModels/incompressible/LES/LESModel/LESModel.H").